backend/djangoindia/db/models/event.py

```diff
@@ -25,7 +25,7 @@ class Event(BaseModel):
     description = models.TextField()
     venue = models.TextField(default="TBA",null=True, blank=True)
     city = models.CharField(max_length=255, default="TBA", null=True, blank=True)
-    venue_map_link = models.TextField(null=True, blank=True)
+    venue_map_link = models.URLField(null=True, blank=True) #will only accepts links with "https://"

```

Comment:
we don't want to support only `https://` links, we are just pasting the venue map's iframe link here which we are simply putting inside an iframe tag in frontend. you can get some idea [here](https://github.com/djangoindia/djangoindia.org/blob/main/frontend/src/containers/Event/Event.tsx#L108-L116).